plugins/datadog/app/models/datadog_deploy_event.rb

```diff
@@ -14,6 +14,18 @@ def self.deliver(deploy, tags:, time:)
         "error"
       end
 
+    tags += ["deploy"]
+
+    kubernetes_template =
+      defined?(SamsonKubernetes) &&
+      deploy.kubernetes? &&
+      deploy.kubernetes_release&.release_docs&.first&.resource_template&.first
+    if kubernetes_template
+      tags << "kube_project:#{kubernetes_template.dig(:metadata, :labels, :project)}"
+      tags << "kube_role:#{kubernetes_template.dig(:metadata, :labels, :role)}"
```

Comment:
how does this work with multiple roles being deployed? need to loop through the resources...

Reply:
yeah that won't work ... we can do project/team first and see if that is good enough
we could tag a single event with all roles that were deployed, but that's kinda fishy / means we have to parse all the docs (5 clusters * 3 roles * 3 resources = 45 resources 🤷‍♂ )